packages/config/config/devices/0x013c/pat02-a.json

```diff
@@ -0,0 +1,338 @@
+			"description": "Determines the type of report sent.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Notification report",
+					"value": 0
+				},
+				{
+					"label": "Sensor binary report",
+					"value": 1
+				}
+			]
+		},
+		"7[0x05]": {
+			"label": "Send Multi CC Reports",
+			"description": "Determines whether a Multi CC report is sent when sending the automatic report.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"7[0x06]": {
+			"label": "Report Battery State When Triggered",
+			"description": "Determines whether the battery state is reported when the device is triggered.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 1
+				},
+				{
+					"label": "Enabled",
+					"value": 0
+				}
+			]
+		},
+		"10": {
+			"label": "Auto Report Battery Time",
+			"description": "The time interval upon which to report the battery level.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"13": {
+			"label": "Auto Report Temperature Time",
+			"description": "The time interval upon which to report the temperature.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"14": {
+			"label": "Auto Report Humidity Time",
+			"description": "The time interval upon which to report the humidity.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"15": {
+			"label": "Auto Report Flood Time",
+			"description": "The time interval upon which to report the flood state.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"20": {
+			"label": "Auto Report Time Unit",
+			"description": "Determines the time unit used by parameters 10 & 13-15 (time interval * time unit).",
```

Comment:
I don't entirely understand how this is supposed to work, but I guess that's what the manual is for

Reply:
Beats me, but that is what it says. It sets units of minutes, on which the others report according to their interval.